### PR TITLE
Fix/ens screen fixes

### PR DESCRIPTION
**Pull request by hydr063n**
make two small fixes to the ENS screen

Before

After

### CODE REVIEW

File: ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml [hunk 1]
```diff
@@ -315,12 +315,12 @@ Item {
...
         anchors.rightMargin: Style.current.padding
-        label: parseFloat(walletModel.getSNTBalance()) < 10 ? 
+        label: parseFloat(utilsModel.getSNTBalance()) < 10 ?
           //% "Not enough SNT"
           qsTrId("not-enough-snt") :
           //% "Register"
           qsTrId("ens-register")
-        disabled: parseFloat(walletModel.getSNTBalance()) < 10 || !termsAndConditionsCheckbox.checked
+        disabled: parseFloat(utilsModel.getSNTBalance()) < 10 || !termsAndConditionsCheckbox.checked
         onClicked: transactionDialog.open()
```
Comment (emizzle) [inline ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml:324]: Maybe while we're here, we could update this to a `StatusButton`? Wdyt?

```qml
StyledButton {
    id: startBtn
    anchors.bottom: parent.bottom
    anchors.bottomMargin: Style.current.padding
    anchors.right: parent.right
    anchors.rightMargin: Style.current.padding
    text: parseFloat(utilsModel.getSNTBalance()) < 10 ?
    //% "Not enough SNT"
    qsTrId("not-enough-snt") :
    //% "Register"
    qsTrId("ens-register")
    enabled: parseFloat(utilsModel.getSNTBalance()) >= 10 && termsAndConditionsCheckbox.checked
    onClicked: transactionDialog.open()
}
```

**Review by emizzle — COMMENTED** [review]
Nice job!

While we're here, could change this button to a `StatusButton`? `disabled` switches to `enabled` (with condition reversed), and `label` changes to `text`. That's it!

File: ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml (hunk 1 above)
Comment (hydr063n) [reply ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml:325]: hey @emizzle,  Sounds good! I also changed the back button into a StatusButton,

**Review by jrainville — APPROVED** [review]
Nice

**Review by richard-ramos — APPROVED** [review]

**Pull request merged**